src/httpclient.c

```diff
@@ -70,6 +70,8 @@ static struct {
 } std_in;
 static int io_interval = 0, req_interval = 0;
 static int ssl_verify_none = 0;
+static int fail = 0;
```

Comment:
Can we use a better name that suggests how this value is used, i.e., exit failure if the server return an error?

Reply:
Indeed. Named it as `exit_failure_on_server_returning_errors`.

Reply:
Thanks. This is much better. I've further changed the name to `exit_failure_on_http_errors`, as we are talking about HTTP status codes indicating errors here.